src/control-plane/backend/lambda/api/service/quicksight/reporting-utils.ts

```diff
@@ -1867,42 +1867,62 @@ export async function warmupRedshift(pipeline: IPipeline, appId: string) {
     return;
   }
   const redshiftType = redshiftEndpoint?.split('.')[3];
-  if (redshiftType === 'redshift-serverless') {
-    const redshiftDataClient = sdkClient.RedshiftDataClient(
-      {
-        region: pipeline.region,
-      },
-      dataApiRole,
-    );
-    const input = {
-      Sqls: [`select * from ${appId}.${EVENT_USER_VIEW} limit 1`],
+  if (redshiftType !== 'redshift-serverless') {
+    return;
+  }
+  const redshiftData = sdkClient.RedshiftData(
+    {
+      region: pipeline.region,
+    },
+    dataApiRole,
+  );
+  const queryId = await waitExecuteWarmupStatement(
+    redshiftData,
+    workgroupName,
+    pipeline.projectId,
```

Comment:
Should we use the database name for the output of the reporting stack if it's available?